core/services/headtracker/head_broadcaster.go

```diff
@@ -20,18 +20,18 @@ type callbackID [256]byte
 
```

Comment:
Kind of weird we use random 256 bytes for this. Why not a UUID?

Reply:
I would rather use just an `int` that would be incremented on each Subscribe. But let it be UUID for now.

Reply:
Please, use the `int` if that is what you think is best